tuf/repository_tool.py

```diff
@@ -39,6 +39,7 @@
 import tempfile
 import shutil
 import json
+import hashlib
```

Comment:
Why use `hashlib` directly instead of `securesystemslib`?

Reply:
Good point. I updated this in the most recent commit.

Reply:
Thanks!